tmk_core/common/report.h

```diff
@@ -190,9 +190,14 @@ typedef struct {
     int8_t  h;
 } __attribute__((packed)) report_mouse_t;
 
+
 typedef struct {
 #if JOYSTICK_AXES_COUNT > 0
-    int8_t axes[JOYSTICK_AXES_COUNT];
+#    if JOYSTICK_AXES_RESOLUTION <= 8
```

Comment:
```suggestion
#    if JOYSTICK_AXES_RESOLUTION == 8
```

Reply:
I can only tell you what I have observed in my testing. I'm flashing a handwired keyboard/joystick gamepad, and observe the joystick output using the Windows "Game Controllers" panel. 
If I use `if JOYSTICK_AXES_RESOLUTION == 8`, then the joystick will _not_ work, if `JOYSTICK_AXES_RESOLUTION `isn't manually defined. 
I'm pretty certain this change would break any current implementation of joysticks in QMK until they add `#define JOYSTICK_AXES_RESOLUTION 8` (or whatever resolution they want) to their code. 
Should I still commit the change?